docs/feature_tap_dance.md

```diff
@@ -196,22 +196,22 @@ SRC += your_name.c
 Pretty simple. It is a nice way to keep some rules common on all your keymaps.
 
 
-### In `/qmk_firmware/users/<your_name>/<you_name>.h`
+### In `/qmk_firmware/users/<your_name>/<your_name>.h`
 
 You will need a few things in this file:
 
 ```c
 #ifndef YOUR_NAME
 #define YOUR_NAME
+#endif
```

Comment:
This `#endif` isn't supported to be here. 

Though, the entire top block could/should be replaced with `#pragma once`.